backend/src/main/java/com/redhat/cloud/notifications/oapi/OApiFilter.java

```diff
@@ -229,34 +226,6 @@ private String filterConstantsIfPresent(String in, String what, String version)
         return null;
     }
 
-    /**
```

Comment:
With the bump of quarkus, `OpenApiTest.validateOpenApi` failed with error `Security scheme not defined in model: SecurityScheme`.
I checked it on Swagger and it didn't compli to its validation rules.
instead of having an almost empty security section, I removed it by adding `quarkus.smallrye-openapi.auto-add-security-requirement=false` config key on `application.properties` file.